src/components/builder.ts

```diff
@@ -609,7 +609,14 @@ export class Builder {
         const regexTexOptions = /^(?:%\s*!\s*T[Ee]X\s(?:TS-)?options\s*=\s*(.*)$)/m
         const regexBibOptions = /^(?:%\s*!\s*BIB\s(?:TS-)?options\s*=\s*(.*)$)/m
         const regexRecipe = /^(?:%\s*!\s*LW\srecipe\s*=\s*(.*)$)/m
-        const content = fs.readFileSync(rootFile).toString()
+        let content = ''
+        for (const line of fs.readFileSync(rootFile).toString().split('\n')) {
+            if (line.match(/^%\s*!/) || line.trim().length === 0) {
```

Comment:
With the current logic, the magic comment must the first non empty line. I think we should make the `!` optional, so that first comment block is composed of consecutive comments (not necessarily magic ones) or empty lines.